models/models.go

```diff
@@ -105,9 +105,12 @@ type StepModel struct {
 	//  steps will run which are marked with IsAlwaysRun.
 	IsSkippable *bool `json:"is_skippable,omitempty" yaml:"is_skippable,omitempty"`
 	// RunIf : only run the step if the template example evaluates to true
-	RunIf   *string                `json:"run_if,omitempty" yaml:"run_if,omitempty"`
-	Timeout *int                   `json:"timeout,omitempty" yaml:"timeout,omitempty"`
-	Meta    map[string]interface{} `json:"meta,omitempty" yaml:"meta,omitempty"`
+	RunIf   *string `json:"run_if,omitempty" yaml:"run_if,omitempty"`
+	Timeout *int    `json:"timeout,omitempty" yaml:"timeout,omitempty"`
+	// The timeout (in seconds) until a Step with no output (stdout/stderr) is aborted
+	// Default: 0; Timeout disabled: -1
```

Comment:
Timeout disabled: -1 -> do we need to handle this case separately?
AFAIK the old timeout is used if > 0. Should not we simply do the same with this timeout?

Reply:
I agree, I think it means the same to have the timeout disabled or not having a timeout at all.

Reply:
Agreed to do not use 0 as the default, so we can differentiate between the value being nil (not set) and disabled (set to 0).